src/safeposix/dispatcher.rs

```diff
@@ -124,9 +122,18 @@ pub extern "C" fn dispatcher(cageid: u64, callnum: i32, arg1: Arg, arg2: Arg, ar
         LSEEK_SYSCALL => {
             cage.lseek_syscall(unsafe{arg1.dispatch_int}, unsafe{arg2.dispatch_isize}, unsafe{arg3.dispatch_int})
         }
+        XSTAT_SYSCALL => {
```

Comment:
Ideally we're trying to order the dispatcher by syscall number, which was why XSTAT was up where it was. We might want to reorganize the numbers later, but let's keep things in numeric order